dace/codegen/targets/fpga.py

```diff
@@ -26,6 +26,16 @@
 from dace.properties import Property, make_properties, indirect_properties
 from dace.symbolic import evaluate
 
+_CPU_STORAGE_TYPES = {
+    dace.dtypes.StorageType.CPU_Heap, dace.dtypes.StorageType.CPU_ThreadLocal,
+    dace.dtypes.StorageType.CPU_Pinned
+}
+_FPGA_STORAGE_TYPES = {
+    dace.dtypes.StorageType.FPGA_Global, dace.dtypes.StorageType.FPGA_Local,
+    dace.dtypes.StorageType.FPGA_Registers,
+    dace.dtypes.StorageType.ShiftRegister
```

Comment:
is ShiftRegister currently supported by other platforms? If not, I'd probably consider changing that to `FPGA_ShiftRegister` until we do... :/

Reply:
Ok, done